### PR TITLE
docs: update readme for ssr package

### PR DESCRIPTION
**Pull request by daine**
# Summary | Résumé
Update the README for the SSR package

Will need a french translation, but would be great to finalize the english version first

### CODE REVIEW

File: packages/ssr/README.md [hunk 1]
```diff
@@ -1 +1,36 @@
...
+
+### Install from npm
+
+Navigate to the root folder of your project and run:
+
+``` js
+npm install @cdssnc/gcds-components @cdssnc/gcds-components-ssr
+// TODO: Review this line above, check if it's correct, or is it just:
+npm install @cdssnc/gcds-components-ssr
+```
```
Comment (daine) [inline packages/ssr/README.md:22]: @ethanWallace would you mind double checking if this instruction is correct? thanks!

Comment (ethanWallace) [reply packages/ssr/README.md:22]: it would be just the bottom one `npm install @cdssnc/gcds-components-ssr`

**Comment by ethanWallace** [comment]
It's looking good so far, I started [this branch](https://github.com/cds-snc/gcds-components/tree/ssr-readme) before that lays out some additional section we will need to have in the readme

**Comment by adorayi** [comment]
Duplicate.

**Comment by daine** [comment]
Closing this as we have 3 separate files/PRs/branches for this task. Keeping this document as the main one for now and will re-open or update the other branch to match instead: https://docs.google.com/document/d/1rEUN53AJuNyg3GjotDl_wCJM-swEDXyqWBJYpBBjTOU/edit?tab=t.0#heading=h.w381uldnna52

**Pull request closed without merging**